src/build_infrastructure/android/source.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
```

Comment:
Oh boy, a `.bat`! Does it really work?

Reply:
Right!!! Not sure if we need it, but since it was in the integration test one, I figure I'd include here as well.